test/clients/firewall_auto_test.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require_relative "../../src/clients/firewall_auto.rb"
```

Comment:
this is problematic as requiring client cause its execution. So we usually avoid it. E.g. all mocking is not done during such run. For auto client I expect that without arguments it do nothing, but you should not rely on it. As e.g. with using https://github.com/yast/yast-yast2/blob/master/library/general/src/lib/installation/auto_client.rb#L60 new base class for auto clients it will raise exception.